### ISSUE TITLE
WPML support

### ISSUE THREAD

**Issue by laustdeleuran** [unfulvio/wp-api-menus#50]
WP-API supports WPML out of the box, so API queries with the parameter `lang=CODE` or similar just works. This doesn't seem to work in this plugin. Are there any plans to implement this support or am I missing some way it's already implemented?

**Comment by isuke01** [comment]
Yes that would be nice, now you must find correct navi ID and get this navi by ID

**Comment by sarahannnicholson** [comment]
Hey @laustdeleuran and @isuke01,
I came across the same thing earlier today! I'm using V2 of this plugin. Here's how I solved it. 
1. In WordPress, localize your English menu [WPML translating menus docs](https://wpml.org/documentation/getting-started-guide/translating-menus/)
2. I created a custom plugin in WordPress. [WordPress plugins tutorial](https://developer.wordpress.org/plugins/intro/)
3. In the my plugin file I added this code:
```
// Add Locale to Menu endpoint		(eg /wp-api-menus/v2/menus/:id )
add_filter('rest_menus_format_menu', 'add_locale_to_menu', 100, 1);
if (!function_exists('add_locale_to_menu')) {
    function add_locale_to_menu($rest_menu) {
		$args = array('element_id' => $rest_menu['ID'], 'element_type' => 'nav_menu' );
		$rest_menu['locale'] = apply_filters( 'wpml_element_language_code', null, $args );
		return $rest_menu;
    }
}
```

**Comment by retzion** [comment]
Would love to use `/wp-api-menus/v2/menus?lang=en` when you get a chance. Is it in the roadmap?
